src/components/reusable/stepper/stepperItem.scss

```diff
@@ -0,0 +1,292 @@
+@use '../../../common/scss/global.scss';
+@use '~@kyndryl-design-system/shidoka-foundation/scss/mixins/typography.scss';
+
+:host {
+  width: 100%;
+}
+p {
+  margin: 0;
+}
+.stepper {
+  &-step {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    //   width: 174px;
+    &-first {
+      align-items: flex-start;
+    }
+    &-last {
+      align-items: flex-end;
+    }
+  }
+  &-icon {
+    text-align: center;
+    &-pending {
+      border: 2px dashed var(--kd-color-spruce-20, #8ecfd9);
+      border-radius: 50%;
+    }
+    &-large {
+      width: 24px;
+      height: 24px;
+    }
+    &-small {
+      width: 16px;
+      height: 16px;
+    }
+  }
+  &-progress {
+    flex: 1 1 auto;
+    height: 1px;
+    overflow: hidden;
+    right: calc(-50% + 20px);
+    left: calc(50% + 20px);
+    position: absolute;
+    background-color: var(--kd-color-background-inverse-hover, #ddf0f3);
+    border-radius: 5px;
+    &-large {
+      top: 11px;
+    }
+    &-small {
+      top: 8px;
+    }
+    &-first {
+      left: 30px;
+    }
+    &-secondlast {
+      right: calc(-100% + 30px);
+    }
+  }
+}
+
+.progressbar {
+  top: 0;
+  left: 0;
+  width: 0;
+  bottom: 0;
+  position: absolute;
+  transition: transform 0.2s linear;
+  transform-origin: left;
+  background-color: var(--kd-color-border-accent-spruce-light, #3fadbd);
+  &-completed {
+    background-color: var(--kd-color-spruce-50);
+  }
+}
+
+.step {
+  &-content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 4px;
+    &-first {
+      align-items: flex-start;
+    }
+    &-last {
+      align-items: flex-end;
+    }
+  }
+
+  &-text {
+    margin-top: 8px;
+    color: var(--kd-color-text-primary);
+    font-family: Roboto;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: normal;
+    text-transform: uppercase;
+    &-disabled {
+      cursor: not-allowed;
+      color: var(--kd-color-darkstone-40);
+      pointer-events: none;
+    }
+    &-error {
+      color: var(--kd-color-error);
+    }
+  }
+
+  &-title-wrapper {
+    display: flex;
+    gap: 4px;
+    align-items: center;
+  }
+
+  &-title {
+    margin-top: 4px;
+  }
+
+  &-title-text {
+    color: var(--kd-color-text-secondary);
+    font-family: Roboto;
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 20px;
+    letter-spacing: 0.16px;
```

Comment:
Needs to use a Foundation typography style.

Reply:
We don't have in designed. I asked Marco earlier.

Reply:
Done changed by Marco